src/prettyprinting.jl

```diff
@@ -124,12 +174,25 @@ end
 
 function Base.show(io::IO, ::MIME"text/plain", ψ::Object{FR}) where {FR<:FusionRing}
     if get(io, :typeinfo, Any) !== Object{FR}
-        print(io, ψ.id, " ∈ 𝒪(", FR, ")")
+        print(io, ψ.id, " ∈ Irr[", FR, "]")
     else
         print(io, ψ.id)
     end
 end
 
+function Base.show(io::IO, ψ::Object)
+    if typeof(ψ) === get(io, :typeinfo, Any)
+        print(io, ψ.id)
+    else
+        print(io, typeof(ψ), "(", ψ.id, ")")
+    end
+    return nothing
+end
+
+function Base.show(io::IO, ::Type{Object{F}}) where {F<:FusionRing}
+    return print(io, "Irr[", F, "]")
+end
```

Comment:
I think this is the only definition that makes me feel slightly uncomfortable, because we don't own `Type` or `show`. This is precisely why Jutho made `type_repr`, which is the pretty-printing version, is there a way to avoid having this definition here?
For example, one way could be to replace the printing of the types in the other show functions to explicitly call `type_repr` instead

Reply:
Redefining `show` for `::Type{MyOwnType}` is indeed a no-go.